From cbb6d080fb0263eaeb2eafe9f7795d8d6a333242 Mon Sep 17 00:00:00 2001
Message-ID: <cbb6d080fb0263eaeb2eafe9f7795d8d6a333242.1718966788.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1718966788.git.sharlatanus@gmail.com>
References: <cover.1718966788.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 21 Jun 2024 08:03:36 +0100
Subject: [PATCH 19/30] gnu: python-stsci-stimage: Update to 0.2.9.

* gnu/packages/astronomy.scm (python-stsci-stimage): Update to 0.2.9.
[native-inputs]: Add python-wheel.

Change-Id: I87595ff5b40506451e69057cde249af0ac6462ea
---
 gnu/packages/astronomy.scm | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 1ea0c8ce7c..bed1d93b83 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3288,16 +3288,17 @@ (define-public python-stsci-stimage
     (name "python-stsci-stimage")
     ;; PyPI version was 0.2.8 but the latest version tag on GiHub was 0.2.7,
     ;; see <https://github.com/spacetelescope/stsci.stimage/issues/38>
-    (version "0.2.8")
+    (version "0.2.9")
     (source (origin
               (method url-fetch)
               (uri (pypi-uri "stsci_stimage" version))
               (sha256
                (base32
-                "0p188my6djy65a8j6qsc3v0dpr2svqqv90n0pxxzy52vdjiz3w0w"))))
+                "18sqmjiyn76hzkmv3g8549vfygi136gnar6pb0s7wb44j7cvc0in"))))
     (build-system pyproject-build-system)
     (arguments
      (list
+      #:build-backend "setuptools.build_meta"
       #:test-flags #~(list "test_c")
       #:phases
       #~(modify-phases %standard-phases
@@ -3312,7 +3313,10 @@ (define-public python-stsci-stimage
     (propagated-inputs
      (list python-numpy))
     (native-inputs
-     (list python-pytest python-setuptools-scm python-waf))
+     (list python-pytest
+           python-wheel
+           python-setuptools-scm
+           python-waf))
     (home-page "https://stscistimage.readthedocs.io/en/latest/")
     (synopsis "STScI image processing")
     (description
-- 
2.41.0

